storage/dial.go

```diff
@@ -60,9 +60,10 @@ func Client(ctx context.Context, config *conf.GlobalConfiguration) (*tigris.Clie
 	defer drv.Close()
 
 	tigrisConfig := &tigris.Config{
-		URL:     config.DB.URL,
-		Project: config.DB.Project,
-		Branch:  config.DB.Branch,
+		URL:      config.DB.URL,
+		Project:  config.DB.Project,
+		Branch:   config.DB.Branch,
+		Protocol: driver.HTTP,
```

Comment:
why http?

Reply:
HTTP's memory leak was fixed in .25 release.